cljfmt/src/cljfmt/core.cljc

```diff
@@ -699,16 +721,32 @@
   (and (or (z/list? zloc) (z/list? (z/up zloc)))
        (some #(matching-form-index? zloc % context) form-indexes)))
 
-(defn align-form-columns [form aligned-forms alias-map]
-  (let [ns-name  (find-namespace (z/of-node form))
-        context  {:alias-map alias-map, :ns-name ns-name}
-        aligned? #(matching-form? % aligned-forms context)]
-    (transform form edit-all aligned? align-columns)))
+(defn align-form-columns
+  ([form aligned-forms alias-map]
+   (align-form-columns form aligned-forms alias-map default-options))
+  ([form aligned-forms alias-map {:keys [align-single-column-lines?]}]
+   (let [ns-name (find-namespace (z/of-node form))
+         context {:alias-map alias-map, :ns-name ns-name}
+         aligned? #(matching-form? % aligned-forms context)]
+     (transform form
+                edit-all
+                aligned?
+                #(align-columns % align-single-column-lines?)))))
 
 (defn realign-form
-  "Realign a rewrite-clj form such that the columns line up into columns."
-  [form]
-  (-> form z/of-node align-columns z/root))
+  "Realign a rewrite-clj form such that the columns line up into columns.
+
+  Options:
+  - :align-single-column-lines? - When true, includes wrapped values in
+    alignment; when false (default), excludes them to prevent excessive
+    padding."
```

Comment:
I'd rather omit the option docs here, as I think it's better to have a central location to document the options. Otherwise we risk two sets of docs getting out of sync.